### ISSUE TITLE
Not setting Temperature and Fan mode when call a preset if Thermostat is off

### ISSUE THREAD

**Issue by eonnet** [bphillips09/Control4-HA-Thermostat#10]
When you call a preset while thermostat is off it just turns on the thermostat HVAC Mode, the temperature and Fan mode stays the same as they were before the thermostat was turned off. If you call the preset again then it sets the temperature and fan mode to a desired level.

**Comment by bphillips09** [comment]
@eonnet Apologies for the delay - is this issue still present?

**Comment by eonnet** [comment]
Yes i think since there was no update to the driver. I didnt tested it i use a workaround where i call the preset again 5 seconds later

**Comment by eonnet** [comment]
Tested it on the latest release, same thing

> @eonnet Apologies for the delay - is this issue still present?

@bphillips09 tested it on the latest release, same thing.

**Comment by bphillips09** [comment]
@eonnet Sorry for the delay again... I know this is still an issue, will get around to fixing it soon hopefully.
